reports/fennec_dashboard/summarize_csv.ipynb

```diff
@@ -0,0 +1,599 @@
+    "def fetch_deduped_pings(sub_range):\n",
+    "    core_pings = get_pings(sc,\n",
+    "                       app=\"Fennec\",\n",
+    "                       doc_type=\"core\",\n",
+    "                       source_version=\"1\",\n",
+    "                       submission_date=sub_range,\n",
+    "                       fraction=1.0)\n",
+    "\n",
+    "    # We don't need the whole ping. Just get the props we want.\n",
+    "    subset = get_pings_properties(core_pings, [\"clientId\",\n",
+    "                                               \"osversion\",\n",
+    "                                               \"profileDate\",\n",
+    "                                               \"meta/submissionDate\",\n",
+    "                                               \"meta/geoCountry\",\n",
+    "                                               \"meta/appUpdateChannel\",\n",
+    "                                               \"meta/Timestamp\",\n",
+    "                                               \"meta/documentId\"\n",
+    "                                              ])\n",
+    "\n",
+    "    # We can (sadly) have duplicated pings. Apply deduping.\n",
+    "    return subset.map(lambda p: (p[\"meta/documentId\"], p))\\\n",
+    "                 .reduceByKey(lambda a, b: a)\\\n",
+    "                 .map(lambda t: t[1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Aggregate the data\n",
+    "\n",
+    "Fields documentation:\n",
+    "* *os_version* - core pings contain the API level, we need to output the codename+version\n",
+    "* *geo* - the country code from the country originating the pings. We're only interested in some countries, other countries are grouped as \"Other\"\n",
+    "* *channel* - the product channel\n",
+    "* *date* - the first day of the aggregated week/month\n",
+    "* *actives* - the number of clients that were active that day. It checked 'org.mozilla.appSessions' before, can we simply count the number of core pings submitted on that day (dedupe by client id!)?\n",
+    "* *new_records* - profile creation date == submission date\n",
+    " * This could not hold due to broken clocks, temporary loss of network, ...\n",
+    "* *d1* - how many clients were active at least once after the profile creation date?\n",
+    "* *d7* - how many clients were active at least once in the 7 days following profile creation?\n",
+    "* *d30* - how many clients were active at least once in the 30 days following profile creation?\n",
+    "* *hours* - session duration in hours. Currently 0.\n",
+    "* *google, yahoo, bing, other* - Currently 0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def get_country(original_country):\n",
+    "    COUNTRIES_OF_INTEREST = set(['US','CA','BR','MX','FR','ES','IT','PL','TR','RU','DE','IN','ID','CN','JP','GB'])\n",
```

Comment:
I'm not sure how smart python is, but it might be better to define `COUNTRIES_OF_INTEREST` as a global instead of redefining it with each function call